package.json

```diff
@@ -5,21 +5,28 @@
     "serve": "vite preview",
     "build": "scripts/build",
     "check": "scripts/check",
-    "test": "echo \"This template does not include a test runner by default.\" && exit 1"
+    "test": "cypress run-ct"
   },
   "type": "module",
   "devDependencies": {
-    "@sveltejs/vite-plugin-svelte": "1.0.0-next.10",
-    "@tsconfig/svelte": "^1.0.10",
-    "@types/node": "^15.12.2",
+    "@cypress/code-coverage": "^3.9.12",
+    "@cypress/vite-dev-server": "^2.2.2",
+    "@sveltejs/vite-plugin-svelte": "^1.0.0-next.36",
+    "@testing-library/cypress": "^8.0.2",
+    "@tsconfig/svelte": "^1.0.13",
     "@typescript-eslint/eslint-plugin": "^4.26.1",
     "@typescript-eslint/parser": "^4.26.1",
+    "cypress": "^9.3.1",
+    "cypress-svelte-unit-test": "git+https://github.com/flakolefluk/cypress-svelte-unit-test.git",
```

Comment:
This is not the best way to define a package, but since the repo from bahmutov is not yet upgraded to Cypress 7 we need to use this fork.
If the library turns out not maintained anymore, we can eventually create our own fork and maintain it.